packages/runtime/container-runtime/src/connectionTelemetry.ts

```diff
@@ -302,6 +325,18 @@ class OpPerfTelemetry {
 				opPerfData: {},
 			});
 		}
+
+		if (message.type == MessageType.NoOp) {
+			// Count the number of no-ops submitted by this client.
+			// The value is reset when we log the OpStats sampled event.
+			this.noOpCount++;
+		}
+
+		this.opsLogger.sendPerformanceEvent({
```

Comment:
This event is sent for all ops? I'm confused, because sampler will zero noOpCount, so that tells me your intention is to sample only noops.

Reply:
I think it's more natural to sample noops themselves, and record how many ops were in between noops (and maybe total content size send in those ops). That should be enough to answer all the questions we might have about noops.

I do not have good mental model how to think about sampling all ops. I.e. I do not know how to get average distance between noops from that info easily.